test/test_workflows.py

```diff
@@ -624,18 +457,6 @@ def test_run_simple_production_success(cli_runner, cleanup, cwl_file, metadata):
             None,
             "InputshouldbeaninstanceofWorkflow",
         ),
-        # The metadata has an unexistent step name
-        (
-            "test/workflows/mandelbrot/description.cwl",
-            "test/workflows/mandelbrot/type_dependencies/production/malformed-wrong-stepname_metadata-mandelbrot_complete.yaml",
-            "Thefollowingstepsaremissingfromthetaskworkflow:{'this-step-doesnot-exist'}",
-        ),
-        # The metadata has an unexistent type
-        (
-            "test/workflows/mandelbrot/description.cwl",
-            "test/workflows/mandelbrot/type_dependencies/production/malformed-nonexisting-type_metadata-mandelbrot_complete.yaml",
-            "Unknownexecutionhooksplugin:'MandelBrotDoesNotExist'",
-        ),
```

Comment:
Is this tested somewhere else? Don't we want to keep such tests?

Reply:
The plugin system is tested in the unit tests, but end-to-end is indeed removed. I would add it back, but using simpler workflows. Do you want me to add it in the PR, or is it better to do in the separate?